polyfills/Number/MAX_SAFE_INTEGER/config.json

```diff
@@ -0,0 +1,22 @@
+{
+	"aliases": [
+		"es6"
+	],
+	"browsers": {
+		"ie": "<12",
+		"ie_mob" : "*",
+		"firefox": "<31",
+		"firefox_mob":"<32",
+		"chrome":"<34",
+		"android": "<5",
+		"safari": "<9",
+		"ios_saf": "<9",
+		"opera": "*",
+		"op_mini": "*",
+		"op_mob": "*",
+		"bb": "*"
```

Comment:
We have it
![image](https://cloud.githubusercontent.com/assets/4225330/18668693/158ea63a-7f37-11e6-9119-122a80b4bc7e.png)